src/server/license/server/api_server.go

```diff
@@ -230,9 +230,15 @@ func (a *apiServer) AddCluster(ctx context.Context, req *lc.AddClusterRequest) (
 	}, nil
 }
 
+func stripSecretFromRequest(req *lc.HeartbeatRequest) *lc.HeartbeatRequest {
+	r := *req
+	r.Secret = ""
+	return &r
+}
+
 func (a *apiServer) Heartbeat(ctx context.Context, req *lc.HeartbeatRequest) (resp *lc.HeartbeatResponse, retErr error) {
 	a.LogReq(nil)
```

Comment:
Can you also update `LogReq` to log the request with the secret redacted?

Reply:
absolutely